src/app/cli/src/coda_commands.ml

```diff
@@ -150,6 +150,13 @@ let replace_proposers keys pks =
     (Keypair.And_compressed_pk.Set.of_list kps) ;
   kps |> List.map ~f:snd
 
+let setup_user_command ~fee ~nonce ~memo ~sender_kp payment_body =
+  let payload =
+    User_command.Payload.create ~fee ~nonce ~memo ~body:payment_body
+  in
+  let payment = User_command.sign sender_kp payload in
```

Comment:
Since this function works for user commands in general, `payment_body` and `payment` should be renamed.